src/Walker.py

```diff
@@ -384,6 +390,15 @@ def visit_In(self, node):
         '''
         return '__contains__'
     
+    def visit_Index(self, node):
+        """
+        @node:ast.ast
+        """
+        var = self.visit(node.value)
+        if var.varType == 'int':
+            return
```

Comment:
I think this should return the subtype of the thing that is being sliced. So if it is a str it would return a str, and if it is a collection then it returns its subtype.